frontend/tests/playwright/msw/request-handler.ts

```diff
@@ -82,37 +115,45 @@ export default class Handler {
              * regardless of location the request is made from.
              */
             '*',
-            (req, res, ctx) => {
-               if (passthrough) return req.passthrough();
-               if (once) return res.once(ctx.status(status), ctx.data(body));
+            customResolver ??
```

Comment:
When do you imagine we would use a custom resolver?

Reply:
I could see it being used when we want to use the values of the `request` to modify the `response` or reduce the number of handler creations needed.

For example, if wanted to validate each user type in a single test, we would need to create each handler that would override the same path.

Instead, we could bake each case into a single custom resolver:

```typescript
server.use(Handler.create({endpoint: '/user', method: 'get'}, pro-test-user-response))
// Run test

server.resetHandlers()
Handler.create({endpoint: '/user', method: 'get'}, ifixit-empl-user-response)
// Run test

server.resetHandlers()
...

// vs.

const customR = (req, res, ctx) => {
	const user = req.body.username

	if(username = 'pro-test-user') {
		return res(
			ctx.json({...defaultUserProps, discountTier: ''})	
	}
	else if (username = 'ifixt-empl-test-user') {}
	else if(username = 'reg-test-user') {}
	else 
	...
}

Handler.create({endpoint: '/user', method: 'get'}, customR)
```

Reply:
Obviously, we could just separate each `user` test into its own `test case` but I am just trying to give an example of how the `customResovler` option could be used.

Reply:
oh cool! That makes sense 👍